index.html

```diff
@@ -659,18 +659,18 @@ <h4>Hashing (eddsa-rdfc-2022)</h4>
 
           <ol class="algorithm">
             <li>
+              Let <var>proofConfigHash</var> be the result of applying the
+              SHA-256 (SHA-2 with 256-bit output) cryptographic hashing algorithm [[RFC6234]]
+              to the <var>canonicalProofConfig</var>. <var>proofConfigHash</var> will be
+              exactly 32 bytes in size.
```

Comment:
Just matching style below:

```suggestion
Let <var>proofConfigHash</var> be the result of applying the
SHA-256 (SHA-2 with 256-bit output) cryptographic hashing algorithm [[RFC6234]]
to the <var>canonicalProofConfig</var>. <var>proofConfigHash</var> will be
exactly 32 bytes in size.
```